docs/settingup.md

```diff
@@ -2041,6 +2039,25 @@ Upload a data file to the Data manager.
 }
 ```
 
+</details><details>
+<summary>disconnectelastic</summary>
+
+## DisconnectElastic action
+
+Disconnect from a QSEoK environment. This action will disconnect open websockets towards sense and events. The action is not needed for most scenarios, however if a scenario mixes "elastic" environments with QSEoW or uses custom actions towards other type of environments, it should be used directly after the last action towards the elastic environment.
```

Comment:
other type**s** of environments

or

another type of environment

Reply:
like the second one better